quizapp/src/main/java/billennium/quizapp/controller/CandidateController.java

```diff
@@ -0,0 +1,33 @@
+package billennium.quizapp.controller;
+
+
+import billennium.quizapp.entity.Candidate;
+import billennium.quizapp.exception.CandidateException;
+import billennium.quizapp.repository.CandidateRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.UUID;
+
+import static billennium.quizapp.controller.ControllerConstants.CANDIDATE;
+import static billennium.quizapp.controller.ControllerConstants.EMAIL_PARAM;
+
+
+@RestController
+@RequiredArgsConstructor
+public class CandidateController {
+
+    private final CandidateRepository candidateRepository;
```

Comment:
oddzielmy warstwe controllera od modelu. Controller powinien mieć wstrzyknięty service. Na pewno w którymś endpoincie bedziemy musieli napisać logike a nie proste wyciaganie z bazy i wtedy wszystkie metody do tego bedziemy mieć w serwisie.

Reply:
to na pewno z tego co pamiętam chcesz póki co że jak jedna implementacja to nie robimy warstwy interfejsu?

Reply:
byłoby super

Reply:
done